Docs/topics/patching/psp.md

```diff
@@ -1,62 +1,49 @@
+We can't help you with this directly, but there are various guides out there you can go out and find. We personally recommend the guides on [pspunk](https://www.pspunk.com/psp-cfw/).
 
 ### Notes on Jailbreaking a PSP
 
-Try to use the latest firmware. We recommend PRO-C over other CFWs, as they have not been thoroughly tested with Allefresher yet. It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, we'd recommend dual-booting PRO-C as well until Allefresher is guaranteed to work well on ARK-4.
-
-### Notes on Jailbreaking a Vita
-
-You'll need to set up Adrenaline with the latest version of the Adrenaline ePSP firmware (as with PSP, ARK-4 is untested). You'll also need [VitaShell](https://github.com/TheOfficialFloW/VitaShell/releases).
-
-## Step 2: Connect your PSP/Vita or Memory Stick to your computer
-
-Connect your console to your computer over USB, or plug your Memory Stick into a card reader.
-
-### Connecting a PSP
-
-Select USB Mode on the XMB.
-
-### Connecting a Vita
-
-Open VitaShell, and press select to enable USB. You may need press start and switch SELECT from FTP -> USB.
-
-Mount the drive and open it in your file explorer.
-
-## Step 3: Setup Refresher
+Try to use the latest firmware (6.61 as of writing). 
+We recommend PRO-C over other CFWs, as they have not been thoroughly tested with Allefresher yet. 
+It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, 
+we'd recommend dual-booting PRO-C as well until Allefresher is guaranteed to work well on ARK-4.
 
-Open Refresher, and select "PSP Setup".
+## Step 2: Go to the browser on your PSP
 
-Inside Refresher, you should see at a dropdown labeled "PSP Drive". In that dropdown, select the correct path. If your console's drive does not appear there, check to make sure it is mounted, and ensure it contains a `PSP` or `pspemu` folder on it.
+<note>If you cannot connect your PSP to your main network, try using a password-less hotspot from your computer or phone.</note>
 
-<note>
-On Windows, the `pspemu` folder is marked as hidden by default. 
-To view it in file explorer, you will need to enable viewing hidden files and system folders.
-</note>
+On your PSP's XMB, select `Network` → `Internet Browser`. 
+Navigate to the `Address` bar at the top of the screen, and type in `http://psp.littlebigrefresh.com:2095`.
+Select `Download Allefresher`, then select `Memory Stick` (if that is unavailable, select `System Storage`). 
+Do the same for `LittleBigRefresh (Domain)` and `LittleBigRefresh (Format)`. 
+This installs the plugin, and sets it up to connect to LittleBigRefresh.
 
-## Step 4: Fill in the server URL
+If you are running PRO-C or ME firmware and cannot edit your plugin list, 
+you can scroll down on the website and download `game.txt` to download a pre-setup config file for Refresh. 
```

Comment:
my original writing here is wrong now that you point it out, i meant to say `Allefresher`, not `Refresh`, since the `game.txt` defines the list of plugins which are loaded on game startup 
the domain/format files are what define the plugin to connect to LittleBigRefresh